spring-integration-core/src/main/java/org/springframework/integration/channel/FluxMessageChannel.java

```diff
@@ -52,11 +52,11 @@
 
 	private volatile boolean upstreamSubscribed;
 
-	public ReactiveChannel() {
+	public FluxMessageChannel() {
 		this(DirectProcessor.create());
 	}
 
-	public ReactiveChannel(FluxProcessor<Message<?>, Message<?>> processor) {
+	public FluxMessageChannel(FluxProcessor<Message<?>, Message<?>> processor) {
```

Comment:
What would be the point of creating a `MessageChannel` for `Mono<Message<?>>` - by definition, in an integration flow, we are wiring a producer and consumer together with a pipe to transfer messages.